### ISSUE TITLE
Rollback to 1.4 not working

### ISSUE THREAD

**Issue by Gitgiddy** [Microsoft/TypeScript#3090]
In VS 2015 RC, I ran into some .ts files build issues after upgrading to the Typescript 1.5 beta (and upgrading the .proj file target to 1.5).  So I decided to roll back, uninstalling the 2015/1.5 from the control panel (the only beta VS version I installed) and reverting the proj file - but now VS 2015 doesn't even seem to have the 1.4 components.  I tried repairing the VS 2015 RC installation but that made no noticable change.  And the 2015 RC 1.4 isn't available as a standalone installer.  Is there a fix besides uninstalling and reinstalling VS 2015 RC?

Thanks!

**Comment by danquirk** [comment]
Paging @paulvanbrenk

**Comment by paulvanbrenk** [comment]
Do you have other versions of VS installed? What are the contents of the following folders:
* C:\Program Files (x86)\Microsoft SDKs\TypeScript
* C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.4
* C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Microsoft\TypeScript

and which version does Microsoft.VisualStudio.LanguageServices.TypeScript.dll have in that last folder?

**Comment by paulvanbrenk** [comment]
Also curious what issues you ran into with 1.5.

**Comment by Gitgiddy** [comment]
I have Visual Studio 2010, 2012, 2013 (and now the RC for 2015).  I have since uninstalled and reinstalled the 2015 RC to restore the Typescript integration, so perhaps those forensic details are gone?  But I do now have SDKs\Typescript with \1.0 and \1.4 and those having the expected localizations, lib.d.tses, tsc.exe, tsc.js, tschost.dll.

Common7\...\ has the full complement, I'm sure now that I reinstalled.  And of course the file version of the LanguageServices dll is 14.14.30.22823 and the assembly version is 14.0.0.0.

With Typescript 1.5 beta, I was getting build errors with my existing project some error I didn't spend any time on about something in lib.d.ts already being defined.  That's all I remember and that may be inaccurate.
